index.html

```diff
@@ -3983,8 +3986,9 @@ <h3>Usage Patterns</h3>
       </p>
 
       <p>
-Privacy risks of <a>credential</a> usage occur when unintended or unexpected
-correlation arises from the presentation of <a>verifiable credentials</a>.
+Privacy risks of <a>verifiable credential</a> usage occur when unintended or
+unexpected correlation arises from the presentation of
+<a>verifiable credentials</a>.
```

Comment:
should probably be credential. Whatever you change it to, make sure the sentence uses the same form. Prefer the broader point being made about "credentials".

Reply:
Fixed.